cache/models/samples.js

```diff
@@ -375,4 +528,49 @@ module.exports = {
       return response;
     });
   },
+
+  /**
+   * Upsert sample in Redis.
+   * @param  {Object} qbObj - Query body object
+   * @param  {Object} logObject - Log object
+   * @param  {String} method - Type of request method
+   * @returns {Promise} - Resolves to upserted sample
+   */
+  upsertSampleInRedis(qbObj, logObject, method) {
```

Comment:
I think I prefer naming these functions *without* the "...InRedis" suffix--the fact that they're in *this* "cache" module and not the "db" module says that already, and without the suffix it sets up an equivalence/interchangeability between the db functions and the redis functions which I like